src/servers/src/prometheus.rs

```diff
@@ -603,6 +615,27 @@ pub async fn labels_query(
     PrometheusJsonResponse::success(PrometheusResponse::Labels(sorted_labels))
 }
 
+async fn get_all_column_names(
+    catalog: &str,
+    schema: &str,
+    manager: &CatalogManagerRef,
+) -> std::result::Result<Vec<String>, catalog::error::Error> {
+    let table_names = manager.table_names(catalog, schema).await?;
+
+    let mut labels = HashSet::new();
```

Comment:
Using HashSet means: there is a chance that duplicated column names happens?

Reply:
Yes, it's allowed to have the same column name in different table